src/lib/BasicDropdown/BasicDropdown.svelte

```diff
@@ -119,6 +119,7 @@
     -moz-appearance: none;
     appearance: none;
     width: 100%;
+    border-radius: 0 !important;
```

Comment:
Do we need the `!important` here? Quick test for me in Safari suggests that this fix works fine without it? Might be better to avoid if possible.